blocks/form/mappings.js

```diff
@@ -1,6 +1,6 @@
 import { loadCSS } from '../../scripts/aem.js';
 
-let customComponents = [];
+let customComponents = ['range', 'my-custom-checkbox', 'fd'];
```

Comment:
can't we just pick these names from the folder custom-components? this is making this tooling brittle. the customer should not need to update this file.

Reply:
mappings.js is run in the browser so I can't really filesync read or write.
Although I can have the arrays generated at build time and triggered in the precommit scripts. Going ahead with this.